internal/network/command_network_link_service_association_list.go

```diff
@@ -0,0 +1,76 @@
+package network
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/errors"
+	"github.com/confluentinc/cli/v3/pkg/examples"
+	"github.com/confluentinc/cli/v3/pkg/output"
+)
+
+func (c *command) newNetworkLinkServiceAssociationListCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "List associations for a network link service.",
+		Args:  cobra.NoArgs,
+		RunE:  c.networkLinkServiceAssociationList,
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: `List associations for network link service "nls-123456".`,
+				Code: "confluent network network-link service association list --network-link-service nls-123456",
+			},
+		),
+	}
+
+	c.addNetworkLinkServiceFlag(cmd)
```

Comment:
We can list service associations with the `status.phase` param. (https://docs.confluent.io/cloud/current/api.html#tag/Network-Link-Service-Associations-(networkingv1)/operation/listNetworkingV1NetworkLinkServiceAssociations)

Are we going to add that in future PR?

Reply:
Yes exactly, that'll be included [here](https://github.com/confluentinc/cli/pull/2554).